src/main/config/messages_template.xml

```diff
@@ -438,409 +440,409 @@ See the accompanying LICENSE file for applicable license.
+    <reason>Found @chunk attribute with value '%1' inside combine chunk.</reason>
     <response>Ignoring chunk operation.</response>
   </message>
 
   <!-- End of Java Messages -->
-    
-  <!-- Start of XSL Messages -->  
+
+  <!-- Start of XSL Messages -->
+
   <message id="DOTX001W" type="WARN">
     <reason>No string named '%1' was found for language '%2'. Using the default language '%3'.</reason>
     <response>Add a mapping between default language and desired language for the string '%1'.</response>
   </message>
-  
+
   <message id="DOTX002W" type="WARN">
-    <reason>The title element or attribute in the ditamap is required for Eclipse output.</reason>
+    <reason>The @title attribute or element in the DITA map is required for Eclipse output.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTX003I" type="INFO">
-    <reason>The anchorref attribute should either reference another dita map or an Eclipse XML TOC file. The value '%1' does not appear to reference either.</reason>
-    <response></response>
+    <reason>The @anchorref attribute should either reference another DITA map or an Eclipse XML TOC file.</reason>
+    <response>The value '%1' does not appear to reference either.</response>
   </message>
-  
+
   <message id="DOTX004I" type="INFO">
-    <reason>Found a navref element that does not reference anything.</reason>
-    <response>The navref element should either reference another dita map or an Eclipse XML file.</response>
+    <reason>Found a &lt;navref&gt; element that does not reference anything.</reason>
+    <response>The &lt;navref&gt; element should either reference another DITA map or an Eclipse XML TOC file.</response>
   </message>
-  
+
   <message id="DOTX005E" type="ERROR">
     <reason>Unable to find navigation title for reference to '%1'.</reason>
-    <response>The build will use '%1' as the title in the Eclipse Table of Contents.</response>
+    <response>Using '%1' as the title in the Eclipse Table of Contents.</response>
   </message>
-  
+
   <message id="DOTX006E" type="ERROR">
-    <reason>Unknown file extension in href="%1".</reason>
-    <response>References to non-DITA resources should set the format attribute to match the resource (for example, 'txt', 'pdf', or 'html').</response>
+    <reason>Unknown file name extension in @href attribute value '%1'.</reason>
+    <response>References to non-DITA resources should set the @format attribute to match the resource (for example, 'txt', 'pdf', or 'html').</response>
   </message>
-  
+
   <message id="DOTX007I" type="INFO">
-    <reason>Only DITA topics, HTML files, and images may be included in your compiled CHM file. The reference to "%1" will be ignored.</reason>
-    <response>To remove this message, you can set the toc="no" or processing-role="resource-only" attribute on your topicref.</response>
-  </message>
-  
-  <!-- As of 20012-06-13, DOTX008W is only called for HTML Help and map2htmtoc -->
```

Comment:
We still have code that calls this, I don't think it can be removed https://github.com/dita-ot/dita-ot/blob/6e5d9ba7d7ce842852d2d6100372deec62360874/src/main/plugins/org.dita.xhtml/xsl/map2htmtoc/map2htmlImpl.xsl#L157